src/vignette/util/thumbnail.clj

```diff
@@ -79,6 +79,7 @@
 
 (defn get-or-generate-thumbnail
   [system thumb-map]
+  (assert-original-mime-type (get thumb-map :original) thumb-map)
```

Comment:
I mean, does it allow `/a/ab/SomeImageWithoutAnExtension` through and try to thumbnail it. Weird edge case, but we do have a lot of images without extensions in CEPH